modules/pel/peltool/peltool.py

```diff
@@ -365,14 +369,47 @@ def listOption(path: str, config: Config, extension: str, rev: bool =False):
             file_list.append(file)  ## create list of file names
         # Only process top level directory
         break
-    file_list.sort(reverse=rev) 
+    file_list.sort(reverse=rev)
+    return root, file_list
+
+def listOption(path: str, config: Config, extension: str, rev: bool = False):
+    root, file_list = getFileList(path, extension, rev)
     final_summary = {}
     for file in file_list:
         eid, summary = extractAndSummarizePEL(os.path.join(root, file), config)
         if eid :
             final_summary[eid] = summary
     print(prettyPrint(json.dumps(final_summary, indent=4) , desiredSpace = 29))
 
+def printPELCount(path: str, config: Config, extension: str):
+    count = 0
```

Comment:
may as well add the function comments here like you did with the other functions.